src/background/install-data-generator.ts

```diff
@@ -40,7 +40,7 @@ export class InstallDataGenerator {
             year: currentDate.getUTCFullYear(),
         };
 
-        this.storageAdapter.setUserData({ [LocalStorageDataKeys.installationData]: this.installationData });
+        this.storageAdapter.setUserData({ [LocalStorageDataKeys.installationData]: this.installationData }).catch(console.error);
```

Comment:
so, we don't want to crash if failed? same applies for other cases

Reply:
We were not doing it before... we may want to do something about it but I think that's not in scope for promisifying the API